backend/dtos/user.py

```diff
@@ -1,34 +1,32 @@
 from pydantic import Field
 
-from backend.dtos.common import BaseDTO
+from backend.dtos.common import BaseModel
 
 EMAIL_REGEX = r"^(\w|\.|\_|\-)+[@](\w|\_|\-|\.)+[.]\w{2,4}$"
 HEX_REGEX = r"#?([\da-fA-F]{2})([\da-fA-F]{2})([\da-fA-F]{2})"
 
 
-class Avatar(BaseDTO):
-    background: str = Field(..., regex=HEX_REGEX, example="#ff0000")
-    emoji: str = Field(..., example="bison")
+class Avatar(BaseModel):
+    background: str = Field(regex=HEX_REGEX, example="#ff0000")
+    emoji: str = Field(example="bison")
 
 
-class UserIn(BaseDTO):
+class UserIn(BaseModel):
     email: str = Field(
-        ...,
         min_length=7,
         max_length=256,
         regex=EMAIL_REGEX,
         example="someuser@gmail.com",
     )
-    password: str = Field(..., min_length=8, max_length=64, example="123456")
+    password: str = Field(min_length=8, max_length=64, example="123456")
 
 
-class UserOut(BaseDTO):
+class UserOut(BaseModel):
```

Comment:
a)Could you explain what for we need class UserOut?

Reply:
that's what will be returned when FE calls BE endpoint to get a user